changelogs/fragments/module-zabbix_template-argspec-doc.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_template - fixed documentation for ``macros`` argument.
```

```suggestion
  - zabbix_template - fixed documentation for ``macros`` argument (https://github.com/ansible-collections/community.zabbix/pull/296).
```
Thank you @D3DeFi for this PR!
Can you please add this PR URL to the end?